From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Mon, 16 Mar 2026 10:31:00 +0800
Subject: man-bashlogout

# DP: document /etc/bash.bashrc in bash man page

# DP: document /etc/bash.bashrc in bash man page
---
 doc/bash.1 | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/doc/bash.1 b/doc/bash.1
index 1c4efd7..ef60897 100644
--- a/doc/bash.1
+++ b/doc/bash.1
@@ -13414,6 +13414,9 @@ The systemwide initialization file, executed for login shells
 .FN /etc/bash.bashrc
 The systemwide per-interactive-shell startup file
 .TP
+.FN /etc/bash.bash.logout
+The systemwide login shell cleanup file, executed when a login shell exits
+.TP
 .FN \*~/.bash_profile
 The personal initialization file, executed for login shells
 .TP
